test-frame-common/src/main/java/io/skodjob/testframe/executor/ExecBuilder.java

```diff
@@ -25,7 +25,9 @@ public class ExecBuilder {
     /**
      * Constructor
      */
-    public ExecBuilder() { }
+    public ExecBuilder() {
```

Comment:
```suggestion
    private ExecBuilder() {
```
:D

Reply:
that cannot be private, as you need to call it from somewhere else

Reply:
I guess you are changing it in your PR or?